order/order.js

```diff
@@ -0,0 +1,37 @@
+let userName=document.querySelector('#userName');
+let addressUser=document.querySelector('#address');
+let phoneUser=document.querySelector('#phone')
+let submitData=document.querySelector('.submit-order');
+let cancelOrder=document.querySelector('.cancel-order');
+let moreDetails=document.querySelector('#moreDetails')
+
+function addInfoUserLocalStorage(){
+    let detailsUser={
+        name:userName.value,
+        address:addressUser.value,
+        phone:phoneUser.value,
+        details:moreDetails.value
+    }
+    localStorage.setItem('detailsUser',JSON.stringify(detailsUser))
+}
+
+
+submitData.addEventListener('click',(e)=>{
```

Comment:
before submit data check if the user enter required fields or not, if he isn't give him a warning but if he entered all of them and clicked submit give him a success pop up message